docs/docs/docs/features/translations/index.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Not sure about this section, but we could keep it for now and removing it later on..

Reply:
Yeah, I decided to keep it because Compoendium, at the moment, supports it.